charts/nginx/values.yaml

```diff
@@ -9,7 +9,7 @@ tolerations: []
 images:
   nginx:
     repository: quay.io/astronomer/ap-nginx
-    tag: 0.33.0
+    tag: 0.43.0
```

Comment:
it's important to test ingress... especially airflow rest api.

Reply:
How do we normally test this? Is there an automated way?

Reply:
similar to https://github.com/astronomer/astronomer/pull/1034#discussion_r591845437 but do some tricky curl quries, i know qa team can do it automatically.